### PR TITLE
Donation Page

### PR DESCRIPTION
**Pull request by xuejingao**

### CODE REVIEW

**Comment by xuejingao** [comment]
image

File: src/pages/donate/index.jsx [hunk 1]
```diff
@@ -1,12 +1,59 @@
...
+		color: THEME_COLOR.Primary,
+		marginTop: "24px",
+	},
+	titles: {
+		color: THEME_COLOR.Secondary,
+	}
+})
 
 const DonatePage = () => {
+	const style_classes = styles()
```
Comment (shikdernyc) [inline src/pages/donate/index.jsx]: NBD: this should be camelCase

Comment (shikdernyc) [reply src/pages/donate/index.jsx]: also probably should follow hooks common syntax `useStyles()`

**Review by shikdernyc — APPROVED** [review]
Nice Design, LGTM.

Minor & NBD: A bit of inconsistent styling like the camel case. Some files are 2 spaces and some are tabs.

**Review by rvente — APPROVED** [review]

**Pull request merged**